packages/fast-components-react-base/src/viewport-positioner/viewport-positioner.tsx

```diff
@@ -303,6 +318,12 @@ class ViewportPositioner extends Foundation<
             (isNil(this.positionerRect) && !this.state.noObserverMode);
 
         return {
+            height: this.props.scaleToFit
+                ? `${this.state.verticalSelectedPositionHeight}px`
+                : null,
+            width: this.props.scaleToFit
+                ? `${this.state.horizontalSelectedPositionWidth}px`
+                : null,
```

Comment:
nit: in order to not do this same query twice you could change this to:
```typescript
return {
    ...(
        this.props.scaleToFit
            ? {
                height: `${this.state.verticalSelectedPositionHeight}px`,
                width: `${this.state.verticalSelectedPositionHeight}px`
            }
            : {}
    ),
    // other properties
}
```

Reply:
yes, better